packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```diff
@@ -28,6 +28,10 @@ spec:
     nodeSelector:
       {{- toYaml . | nindent 6 }}
   {{- end }}
+  {{- with .Values.fluentbit.resources }}
```

Comment:
Where is fluentbit.resources set ? I don't see it in values.yaml

Reply:
Ah that was an oversight. Good catch.

Reply:
Oh wait no, it's there. Line 97.

Reply:
I moved them together for readability.